exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h

```diff
@@ -21,6 +22,11 @@ namespace exporter
 namespace otlp
 {
 
+inline const std::string GetOtlpDefaultUserAgent()
+{
+  return "OTel OTLP Exporter Cpp/" OPENTELEMETRY_SDK_VERSION;
```

Comment:
Avoiding including space, like "OtlpExporterCpp/1.2.3"? "OTel OTLP" seems a little redundant as "OTel" is the "O" in "OTLP".

Reply:
Saw the comment https://github.com/open-telemetry/opentelemetry-specification/pull/2684/#discussion_r975520619. The current format looks good.